comms/storage/decider/StorageDecider.go

```diff
@@ -24,3 +38,11 @@ func (d *NaiveDecider) ShouldStore(id string) bool {
 func (d *NaiveDecider) OnChange(prevBuckets []string, curBuckets []string) error {
 	return nil
 }
+
+func (d *NaiveDecider) GetTempStoreFor(bucketOrId string) (nats.ObjectStore, error) {
+	// No sharding for naive decider - store it all in one bucket.
+	return d.jsc.CreateObjectStore(&nats.ObjectStoreConfig{
```

Comment:
I think I had a situation where using `CreateObjectStore` frequently (like in a request handler) caused problems where `ObjectStore` did not...

might just be my imagination, but we should be aware of it.  If problems arise we should pre-create the buckets and just do a get here.

Reply:
that's a good point, and I'm happy to pay that tech debt now - changed this to pre-create ✅